src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Marshalling/MarshallerHelpers.cs

```diff
@@ -394,5 +394,17 @@ public static MarshalDirection GetMarshalDirection(TypePositionInfo info, StubCo
             }
             throw new UnreachableException("An element is either a return value or passed by value or by ref.");
         }
+
+        /// <summary>
+        /// Returns whether a parameter has MIDL '[out]' behavior should be unmarshalled into a local variable and only assigned to the parameter at the end of the function call.
+        /// </summary>
+        public static bool IsMidlOutBehavior(TypePositionInfo info, StubCodeContext context)
```

Comment:
Is there another name we could use here instead of `MidlOut`? I'd rather not include references to MIDL here, especially at this layer where code isn't COM-specific.